test/helpers/kubectl.go

```diff
@@ -2247,6 +2247,18 @@ func (kub *Kubectl) DeleteCiliumDS() error {
 	return kub.waitToDelete("Cilium", CiliumAgentLabel)
 }
 
+func (kub *Kubectl) DeleteHubbleClientPods() error {
+	ginkgoext.By("DeleteHubbleClientPods(namespace=%q)", CiliumNamespace)
```

Comment:
This assumes that the hubble pods are deployed on the cilium namespace. However, the hubble tests in `test/k8sT/hubble.go` seem to leave the option of running them in a different namespace as they define a hubble namespace (currently set to `helpers.CiliumNamespace` though). Should these helpers be updated to take a namespace as parameter?

Reply:
We are not really consistent with the CiliumNamespace. Many Cilium helpers have the same problem, where sometimes it's a parameter, sometimes it's assumed to `helpers.CiliumNamespace`.

But I guess since hubble-cli and relay could in theory be deployed in a non-cilium namespace, I can make it a parameter.